assets/airports/kpvd.json

```diff
@@ -0,0 +1,6504 @@
+{
+    "radio": {
+        "twr": "Providence Tower",
+        "app": "Providence Approach",
+        "dep": "Providence Departure"
+    },
+    "icao": "KPVD",
+    "iata": "PVD",
+    "magnetic_north": -14.5,
+    "ctr_radius": 80,
+    "ctr_ceiling": 12000,
```

Comment:
An SOP I found for VATSIM said the ceiling was 10,000ft, but it was somewhat old. Just curious as to where you got this

Reply:
Not fully sure, but I'd rather stick to 12000' than 10000'

Reply:
It's been a while

Reply:
> I'd rather stick to 12000' than 10000'

We should go for whatever the most realistic option is. The only source I found said 10000 so I'd go for that.